broker/src/main/java/org/apache/rocketmq/broker/out/BrokerOuterAPI.java

```diff
@@ -86,7 +86,7 @@ public void shutdown() {
     public String fetchNameServerAddr() {
         try {
             String addrs = this.topAddressing.fetchNSAddr();
-            if (addrs != null) {
+            if (addrs != null && !UtilAll.isBlank(addrs)) {
```

Comment:
It would be better to combine one with the following if.

Reply:
updated